### PR TITLE
line length change to 88 chars (ruff default)

### LINKED ISSUE #830: line length decision
#829 moves the project to ruff linter. As a side effect we noticed we have quite a few lines that are over 80 characters long -- apparently black is more lenient in many cases. 

That PR will ignore line-too-ling (E501) to keep things moving. Let's decide what to do here:
* We can just fix the errors (either split lines or ignore whole sub modules like tests)
* or we could just use the ruff default 88 chars (and keep ignoring E501 for tests): that looks fine to me and only leaves us with half a dozen lines to fix manually

for context: There are 181 line-too-long errors (125 of them in actual sources, 56 in tests).

-- 

I think I'd be fine with ruff default line length -- it seem to be widely used in the ecosystem.

### PR DESCRIPTION
**Pull request by L77H**
**Description of the changes being introduced by the pull request**:
- line-length specification removed from pyproject.toml
- code changes in `securesystemslib/` & `tests/` to comply with linter
- `# ruff: noqa: E501` added to all test files to be ignored by the linter when checking for line-too-long

format checks are still being performed on `tests/` only checks for line-too-long are omitted on all test files.

Fixes #830

### CODE REVIEW

File: tests/__init__.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 with the '-m' flag, when invoked individually.
 
 """
+# ruff: noqa: E501
```
[the remark this answers is not in the document]
Comment (L77H) [reply tests/__init__.py]: Completely agree, this is much better, I made the change.

**Comment by L77H** [comment]
Thanks @jku! I would prefer to have #836 merged first.

**Review by jku — APPROVED** [review]
LGTM, thank you

**Comment by jku** [comment]
squash merged to avoid several merge commits

**Pull request merged**